docker-compose.memory-leak.yml

```
@@ -3,7 +3,20 @@ services:
+      - MEMLAB_SKIP_WARMUP=true
+      - NEXT_PUBLIC_CONTINUOUS_DEPLOYMENT_HEADER_NAME=no-aws-header-name
+      - NEXT_PUBLIC_CONTINUOUS_DEPLOYMENT_HEADER_VALUE=no-aws-header-value
     init: true
-    shm_size: '256m'
+    shm_size: '1gb'
+    healthcheck:
+      interval: 10s
+      timeout: 5s
+      retries: 3
+      start_period: 30s
+      test: [ "CMD", "chromium-browser", "--version" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)